EnglishTranslation-master/QUEST_LV_0200.tsv

```diff
@@ -189,7 +189,7 @@ QUEST_LV_0200_20150317_000188	Lots of things happend here, but since they occurr
 QUEST_LV_0200_20150317_000189	Everything is being destroyed as time passes by.
 QUEST_LV_0200_20150317_000190	Thank you. This is about the collapse of the fortress.{nl}I should look more into it to find more decisive information.
 QUEST_LV_0200_20150317_000191	I am doing many investigations here, but I am having a hard time discovering an incident that occurred more in the past due to those monsters.{nl}Can you go and look into the past on behalf of me?
-QUEST_LV_0200_20150317_000192	Use the magnifying glass of time near Pergele Roads. Then you would be able see the past.
+QUEST_LV_0200_20150317_000192	Use the magnifying glass of time near Pergales Highway. Then you would be able see the past.
```

Comment:
`Then you would be able see the past.`
to
`You should be able to see the past.`